kuzzleio/sdk-javascript
# Add-getting-started-vuejs

## What does this PR do?

This getting-started has been added to the documentation before but removed because vuepress can't build with it. 

It was approved in those PR : #405 and #406 

Add VueJS getting started without vuex to SDK JS6 documentation.
Resolve the build problem of the vuepress doc (Vuepress tried to build .vue files of the getting-started and crash) by adding symlink.

:warning:  need to add following line in doc/6/getting-started/vuejs/standalone/index.md line 14 after merge: 
``` You can find the full code of this guide [here](https://github.com/kuzzleio/sdk-javascript/tree/6-dev/doc/6/getting-started/vuejs/standalone).```:warning: 

### How should this be manually tested?
See documentation: 
  - Step 1 : npm run doc prepare
  - Step 2 : npm run doc-dev

Test getting started project (You'll need the CYPRESS_RECORD_KEY as environment variable)
  - Step 1 : Get a Kuzzle running without anonymous restrictions
  - Step 2 : cd doc/6/getting-started/vuejs && npm ci
  - Step 3 : npm run serve-standalone
  - Step 4 : npm run test

### Other changes
/

### Boyscout
/
